extensions/src/main/java/cm/aptoide/pt/extensions/DateExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package cm.aptoide.pt.extensions
+
+import java.util.Date
+import java.util.Locale
+import android.icu.text.DateFormat
+
+object DateExtensions {
```

Comment:
What for do we need this object wrapping the extension function? 🤔

Reply:
my bad. fixed